compositor/main.c

```diff
@@ -2768,6 +2729,392 @@ read_rdp_config_int(char *config_name, int default_value)
 	return default_value;
 }
 
+static bool
```

Comment:
I think these new additions to be placed at separate file outside of main.c, like rdpdisp.c (similar idea as rdpaudio.c).

Reply:
Good idea, I've moved these now.